gateway/1.4/modules/ROOT/pages/flex-view-replica-status.adoc

```diff
@@ -5,21 +5,49 @@ endif::[]
 :page-aliases: runtime-manager::flex-gateway-replica-status.adoc
 :imagesdir: ../assets/images
 
-To view information about the status of Flex Replicas:
+In Runtime Manager, you can view the status of groups of Flex Replicas and individual Flex Replicas.
+
```

Comment:
Would it be better to say "View the status of the Flex Gateways (Groups of Flex Replicas)". 
Basically call it Gateway, as it is the the Gateways page you are referring to here. Also **G**roups rather than **g**roups